app/presenters/archived_tag_presenter.rb

```diff
@@ -3,18 +3,34 @@ def initialize(tag)
     @tag = tag
   end
 
+  def draft?
```

Comment:
I'm easy, but the reason I didn't is because you need to defing a tag method (or attr_reader) for this to work.